docs/pages/play.vue

```diff
@@ -257,13 +285,85 @@ export default {
       // Check if editors contain default JS and Template
       return this.js.trim() === defaultJS.trim() && this.html.trim() === defaultHTML.trim()
     },
+    codepen_data() {
+      const data = {
+        editors: '101',
+        layout: 'left', // left, right, top
+        html_pre_processor: 'none',
+        css_pre_processor: 'none',
+        css_prefix: 'autoprefixer',
+        js_pre_processor: 'babel',
+        head: '<meta name="viewport" content="width=device-width">',
+        css_external: [
+          `//unpkg.com/bootstrap@${bootstrapVersion}/dist/css/bootstrap.min.css`,
+          `//unpkg.com/bootstrap-vue@${bootstrapVueVersion}/dist/bootstrap-vue.css`
+        ].join(';'),
+        js_external: [
+          '//unpkg.com/babel-polyfill/dist/polyfill.min.js',
+          `//unpkg.com/vue@${vueVersion}/dist/vue.min.js`,
+          `//unpkg.com/bootstrap-vue@${bootstrapVueVersion}/dist/bootstrap-vue.js`
+        ].join(';'),
+        html: this.fiddle_html,
+        js: this.fiddle_js,
+        css: 'body { padding: 1rem; }'
+      }
+      return JSON.stringify(data)
+    },
+    codesandbox_data() {
+      const html = this.html.trim() || '<div></div>'
+      const css = 'body { padding: 1rem; }'
+      const js = this.js.trim() || '{}'
+      const vueContent = dedent`
+        <template>
+          ${html}
+        <\/template>
+
+        <style>
+          ${css}
+        <\/style>
+
+        <script>
+          export default ${js}
+        <\/script>
+      `.replace(/\\\//g, '/')
+      const htmlContent = '<div id="app"></div>'
+      const jsContent = dedent`
+        import Vue from 'vue'
+        import BootstrapVue from 'bootstrap-vue'
+        import App from './App'
+
+        import 'bootstrap/dist/css/bootstrap.css'
+        import 'bootstrap-vue/dist/bootstrap-vue.css'
+
+        Vue.use(BootstrapVue)
+
+        new Vue({
+          el: '#app',
+          render: h => h(App)
+        })
+      `
+      const dependencies = {
+        bootstrap: bootstrapVersion,
+        'bootstrap-vue': bootstrapVueVersion,
+        'popper.js': popperJSVersion,
```

Comment:
Is this only included because Bootstrap has it as a dependency?

Reply:
You're right, we don't need that in the dependencies here.